tests/test_ecs.py

```diff
@@ -532,43 +533,99 @@ async def test_image_in_task_definition(aws_credentials):
     ), "The prefect container should use the image field"
 
 
-@pytest.mark.parametrize(
-    "task_definition",
-    [
-        # Empty task definition
-        {},
```

Comment:
Looks like we lost the empty task definition case. Should that case still be covered?

Reply:
Yes please!